packages/plugin-ext/src/main/browser/view/plugin-view-widget.ts

```diff
@@ -67,12 +69,14 @@ export class PluginViewWidget extends Panel implements StatefulWidget {
     storeState(): PluginViewWidget.State {
         return {
             label: this.title.label,
+            message: this.message,
```

Comment:
it has to be a part of `TreeViewWidget` state

Reply:
I can do so, but the question is the same - how to access it during the tree view creation, when `TreeViewWidget` is not added to `PluginViewWidget` yet?